### ISSUE TITLE
set timestep manually?

### ISSUE THREAD

**Issue by friforever** [MDAnalysis/mdanalysis#676]
when reading amber netcdf trajectory, timestep is always abnormal, can I set the timestep manually instead of getting a warning says "Reader has no dt information, set to 1.0 ps"?

**Comment by orbeckst** [comment]
Hi @friforever , can you please note which version of MDAnalysis you are using?

I can reproduce the problem with one of the bundled test trajectories, i.e.
```python 
import MDAnalysis as mda
from MDAnalysisTests.datafiles import PRM7, NCDFtruncoct

u = mda.Universe(PRM7, NCDFtruncoct)
print(u.trajectory.dt)
```
and get
```

/Users/oliver/.virtualenvs/mdanalysis/develop/lib/python2.7/site-packages/MDAnalysis/coordinates/base.py:725: UserWarning: Reader has no dt information, set to 1.0 ps
  warnings.warn("Reader has no dt information, set to 1.0 ps")
```

**Comment by orbeckst** [comment]
@richardjgowers it would be good to get a fix into 0.14 if possible. Otherwise this can be a 0.14.1.

**Comment by orbeckst** [comment]
And I *think* the test trajectory should have time step information. Could someone confirm or disprove this assumption?

**Comment by richardjgowers** [comment]
So to answer the question as asked, it should be possible to set the dt by specifying it at Universe creation
`u = mda.Universe(top, trj, dt=100.0)`

But I've also fixed (#687) that the NCDF Reader didn't calculate dt (and only read time)
